script/Ciel/Day 11/11_01_CIEL11_1.txt

```diff
@@ -6,22 +6,23 @@ A hot summer day.
 [sha:54a68f2f4f392134a075af0306c6c302423563f3]{
 -- Page 1, Offset 28972.
 -- 青い空と　大きな大きな入道雲。
-A blue sky with huge, towering cumulonimbus clouds.
+The blue sky.
+ Large, giant clouds towering above.
 }
 [sha:c12e997c9c8593402cca860e0f4c8623cb0ab971]{
 -- Page 1, Offset 28973.
 -- じりじりとゆらぐ風景と
-The landscape shimmers in the heat.
+The landscape shimmering from the heat.
 }
 [sha:3c325a238b489ee153bab2a33827624f7f5b2062]{
 -- Page 1, Offset 28974.
 -- 気が遠くなるような蝉の声。
-and the overwhelming cries of the cicadas.
+The overwhelming cries of cicadas.
```

Comment:
This one needs to be an override, or the matching sentence pair in Arc 9_1 needs to change